Makefile

```diff
@@ -5,17 +5,22 @@ DOCKER ?= docker
 
 COMMIT_NO := $(shell git rev-parse HEAD 2> /dev/null || true)
 GIT_COMMIT := $(if $(shell git status --porcelain --untracked-files=no),${COMMIT_NO}-dirty,${COMMIT_NO})
+GIT_TAG    ?= $(shell git describe 2> /dev/null)
 GIT_BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null)
 GIT_BRANCH_CLEAN := $(shell echo $(GIT_BRANCH) | sed -e "s/[^[:alnum:]]/-/g")
 
-IMAGE_NAME_INIT ?= quay.io/iovisor/kubectl-trace-init
-IMAGE_NAME ?= quay.io/iovisor/kubectl-trace-bpftrace
+GIT_ORG ?= iovisor
+
+IMAGE_NAME_INIT ?= quay.io/$(GIT_ORG)/kubectl-trace-init
+IMAGE_NAME ?= quay.io/$(GIT_ORG)/kubectl-trace-bpftrace
 
 IMAGE_TRACERUNNER_BRANCH := $(IMAGE_NAME):$(GIT_BRANCH_CLEAN)
 IMAGE_TRACERUNNER_COMMIT := $(IMAGE_NAME):$(GIT_COMMIT)
+IMAGE_TRACERUNNER_TAG    := $(IMAGE_NAME):$(GIT_TAG)
```

Comment:
might also want to define `IMAGE_TRACERUNNER_LATEST`

It is referenced here https://github.com/iovisor/kubectl-trace/pull/135/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L81 but seems to be missing definition.

Reply:
huh you're right. Yeah i'll add that.